lib/studies/reply/mail_view_page.dart

```diff
@@ -19,24 +19,24 @@ class MailViewPage extends StatelessWidget {
       body: SafeArea(
         bottom: false,
         child: Container(
-          color: Theme.of(context).cardColor,
-          child: Padding(
-            padding: const EdgeInsetsDirectional.only(
-              top: 42,
-              start: 20,
-              end: 20,
-            ),
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                _MailViewHeader(
-                  email: email,
-                ),
-                const SizedBox(height: 32),
-                Expanded(
-                  child: _MailViewBody(message: email.message),
-                ),
-              ],
+          height: double.infinity,
```

Comment:
Why is this needed?

Reply:
It's needed for the `Container` to expand the whole height of the screen. Without it doesn't fill the screen.